src/main/scala/org/ergoplatform/nodeView/mempool/ErgoMemPool.scala

```diff
@@ -114,6 +134,8 @@ object ErgoMemPool {
 
     case object Accepted extends ProcessingOutcome
 
+    case class DoubleSpendingLoser(winnerTxIds: Set[ModifierId]) extends ProcessingOutcome
```

Comment:
Please add ScalaDoc for all public classes.

Reply:
done